no-computer.md

```diff
@@ -6,39 +6,75 @@ This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
 
+
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+The basis for this language is West Coast Swing (ex: https://youtu.be/QiB-xYd9PSo?t=30), 
+a type of social dancing with specific rules and technique. This can be explained as a 
+DSL by translating those rules into a specific wording that results in particular steps 
+done in a certain way. This helps anyone who wants to dance West Coast Swing to break down 
+the movements. This is more aimed at beginner level dancers than advanced
+dancers who have master most of the movement. By having this broken down as a
+language, it can help beginning dancers organize the different movements they
+have available to them and when it is appropriate to use them.
 
```

Comment:
This seems almost identical to your first idea, just with a specific dance instead of any dance a user wants, which I guess does narrow the domain in a good way.